submissions/webdevtoday/friends-app/js/components/RandomUserAPI.js

```diff
@@ -0,0 +1,9 @@
+export async function getUsers() {
+    const API_URL = new URL('https://randomuser.me/api/');
+    API_URL.searchParams.set('inc', 'gender,name,location,email,registered,dob,picture,phone');
+    API_URL.searchParams.set('noinfo', '');
+    API_URL.searchParams.set('results', '100');
+    API_URL.searchParams.set('seed', 'msfa');
```

Comment:
do you really need `new URL` and `searchParams` here? 
why isn't a template string enough?

Reply:
It's convenient for me. When I was testing it I often changed url params. But if you need to remove it, then I will.

Reply:
no, in this case it is not necessary

Reply:
Ok